examples/gltf/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "gltf"
```

Comment:
Hmmm, `cargo deny` is upset as it thinks this is the same crate as the proper `gltf` one. I wonder if we should change the example crate names, or just not care.

Reply:
Could you add this as an exemption here: https://github.com/BVE-Reborn/rend3/blob/trunk/deny.toml#L16. Then we'll be good to go